aict_tools/configuration.py

```diff
@@ -105,7 +105,7 @@ def from_yaml(cls, configuration_path):
 
     def __init__(self, config):
         self.has_multiple_telescopes = config.get('multiple_telescopes', False)
-        self.coordinate_transformation = config.get('coordinate_transformation', 'FACT')
+        self.coordinate_transformation = config.get('coordinate_transformation', None)
```

Comment:
The exception should be here, not in `train_disp`.

If None, raise an exception like `Required value ... is missing`, if unknown, the exception from below.

Reply:
I was debating that because its not needed for energy or gamma/hadron tasks.
Might be better put it into the DISP config now that I think about it

Reply:
If no disp config is given, this point won't be reached.

Reply:
Ah, sorry. You put it in the general config.

Yes, this should go into the disp config.